in_toto/runlib.go

```diff
@@ -249,7 +249,7 @@ return value is an empty Metablock and the second return value is the error.
 NOTE: Currently InTotoRun cannot be used to sign Link metadata.
 */
```

Comment:
Isn't this note outdated when this PR is in? :)

Reply:
Yep this note will be outdated and should be removed :D

Reply:
removed via dedicated commit